tools/ARIAtools/extractProduct.py

```diff
@@ -1195,7 +1195,10 @@ def export_products(
 
             # track valid files
             prev_outname = os.path.abspath(
-                os.path.join(workdir, i, product_dict[1][0][0]))
+                os.path.join(workdir,
+                             i.split('_')[-1],
```

Comment:
Can we move `i.split('_')[-1]` out of the `os.path.join` expression and into its own expression (i.e. `tag = i.lstrip('_')` then include `tag` in `os.path.join`? Then include a comment about what this is doing (removing leading underscore)

Reply:
Done!